src/DataProvider/ChainItemDataProvider.php

```diff
@@ -35,16 +35,21 @@ public function __construct(array $dataProviders)
     /**
      * {@inheritdoc}
      */
-    public function getItem(string $resourceClass, $id, string $operationName = null, array $context = [])
+    public function getItem(string $resourceClass, $id, string $operationName = null, array $context = []/**, array $identifiers = [] **/)
```

Comment:
IMO we should just normalize the identifiers directly in the `ItemDataProvider` (or use the context), but I would like to get rid of this extra parameter.